share/functions/realpath.fish

```diff
@@ -6,8 +6,10 @@
 # If we see that there is an external command by that name we just return. That will cause fish to
 # run the external command. On the other hand, if an external command isn't found we define a
 # function that will provide fallback behavior.
-if not type -q -P realpath
-    function realpath --description 'fallback realpath implementation'
-        builtin fish_realpath $argv[-1]
-    end
+if not command -v realpath > /dev/null
+	if command -v grealpath > /dev/null
+		alias realpath=grealpath
+	else
+		alias realpath=fish_realpath
```

Comment:
The manpage says specifically to not use `fish_realpath` directly, because we provide an interface through the function "realpath" which will look for an external command else fallback to `fish_realpath`. But you don't like the change here to actually find an external command in a way that works on BSD where it will commonly be installed as `grealpath`?

I mean, if you want to double down on the builtin - I'd rather remove it - I guess you should rename it to just "realpath" and prefer it over BSD realpath as well. 

I'd prefer removing the builtin entirely - I won't b expending a bunch of energy on it - we use it nowhere in our project and searching github for fish script with realpath in it nets few results. It looks like it was implemented rather hastily after one guy filed an issue - what's the bar for a fish builtin?